packages/components/src/VirtualizedList/RowCollapsiblePanel/RowCollapsiblePanel.component.js

```diff
@@ -0,0 +1,90 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+import { CellMeasurer } from 'react-virtualized';
+import CollapsiblePanel from '../../CollapsiblePanel/CollapsiblePanel.component';
+import { extractSpecialFields, getCellData, getId, getRowData } from '../utils/gridrow';
+
+import withListGesture from '../../Gesture/withListGesture';
+import css from './RowCollapsiblePanel.scss';
+
+const DEFAULT_HEIGHT = 50;
+
+/**
+ * Row renderer that displays a Collapsible Panel
+ */
+class RowCollapsiblePanel extends React.Component {
+	render() {
+		const { className, index, onKeyDown, parent, style } = this.props;
+		const { titleField } = extractSpecialFields(parent);
+
+		const parentId = getId(parent);
+		const id = parentId && `${parentId}-${index}`;
+		const rowData = getRowData(parent, index);
+
+		let onRowDoubleClick;
+
+		if (parent.props.onRowDoubleClick) {
+			onRowDoubleClick = event => parent.props.onRowDoubleClick({ event, rowData });
+		}
+
+		const onToggle = (event, measure) => {
+			if (parent.props.onRowClick) {
+				parent.props.onRowClick({ event, rowData: { ...rowData, index } });
+				setTimeout(measure, 0);
+			}
+		};
+
+		return (
+			// eslint-disable-next-line jsx-a11y/no-static-element-interactions
+			<CellMeasurer
+				cache={parent.props.deferredMeasurementCache}
+				columnIndex={0}
+				key={index}
+				parent={parent}
+				rowIndex={index}
+			>
+				{({ measure }) => (
+					<div
+						className={classNames(css['tc-collapsible-row'], rowData.className, className)}
+						id={id}
+						onDoubleClick={onRowDoubleClick}
+						onKeyDown={e => onKeyDown(e, this.ref)}
+						ref={ref => {
+							this.ref = ref;
+						}}
+						role="listitem"
+						tabIndex="0"
+						aria-posinset={index + 1}
+						aria-setsize={parent.props.rowCount}
+						aria-label={titleField && getCellData(titleField, parent, index)}
```

Comment:
This is always undefined. You don't pass any column definition so no title column, so the `extractSpecialFields` doesn't work here.
You need to use the panel info to get the title.